docs/modify-integration.md

```diff
@@ -0,0 +1,24 @@
+## Modifying built-in integrations
+Sematext App Agent has built-in support for various application [integrations](https://github.com/sematext/sematext-agent-integrations). 
+You can modify the built-in integration YAMLs to suit your needs or fix any issues present in these YAMLs.
+These changes can be modifying existing YAML files ( for example, adding new metric or modifying the description of existing metric) 
+or adding a new YAML file. Assuming you have installed Sematext App Agent (open source or enterprise version) and
```

Comment:
What is the "enterprise version" of the App Agent?

Reply:
This is the spm-client package which includes app agent + STA.

Reply:
Aha.  A bit strange naming (because we don't use that name anywhere).
Maybe: Assuming you have installed Sematext App Agent (by building it yourself or by installing the spm-client package from Sematext Cloud)

Reply:
done